lib/resty/upstream/healthcheck.lua

```diff
@@ -193,11 +205,40 @@ local function peer_ok(ctx, is_backup, id, peer)
         peer.down = nil
         set_peer_down_globally(ctx, is_backup, id, nil)
     end
+
+    local u_key = gen_upstream_key(u,"all_down")
```

Comment:
Please always add a space after commas (`,`). There are other places having the same style issue.

Reply:
Fixed.